plugins/context-plugin.js

```diff
@@ -4,34 +4,37 @@
 This is where new ctx properties should be initialized
 */
 
-const envVarsPlugin = require('./environment-variables-plugin');
-const compilationMetaDataPlugin = require('./compilation-metadata-plugin');
+const getEnv = require('./environment-variables-plugin');
```

Comment:
Thoughts on removing `plugin` from the names here, to differentiate from the DI system?

Reply:
yea that sounds good